Source/Math/NcclComm.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2016, NVIDIA CORPORATION. All reights reserved.
```

Comment:
Please also add this line below:
// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.